azure-application-insights-spring-boot-starter/src/main/java/com/microsoft/applicationinsights/autoconfigure/ApplicationInsightsTelemetryAutoConfiguration.java

```diff
@@ -163,7 +170,22 @@ public TelemetryClient telemetryClient(TelemetryConfiguration configuration) {
 
     @Bean
     @ConditionalOnMissingBean
-    public TelemetryChannel telemetryChannel() {
+    public TelemetryChannel telemetryChannel() throws IllegalConfigurationException {
+        String localForwarderEndpoint = environment.getProperty("azure.application-insights.channel.local-forwarder.endpoint-address");
+        String inProcessEndPoint = environment.getProperty("azure.application-insights.channel.in-process.endpoint-address");
+        if (StringUtils.isNotBlank(localForwarderEndpoint) && StringUtils.isNotBlank(inProcessEndPoint)) {
+            throw new IllegalConfigurationException("SDK cannot have two channels, please either remove Local Forwarder Endpoint, "
+                + "or In Process Endpoint");
+        }
+
+        // If local forwarder endpoint is present configure local forwarder channel
+        if (StringUtils.isNotBlank(localForwarderEndpoint)) {
+            LocalForwarder lf = applicationInsightsProperties.getChannel().getLocalForwarder();
+
+            // only the endpoint address is picked up. All the other values are fake and just for future use if needed.
+            return new LocalForwarderTelemetryChannel(lf.getEndpointAddress(), false, 0 ,0);
```

Comment:
the last two arguments should be configurable. see [this constructor](https://github.com/Microsoft/ApplicationInsights-Java/blob/d7b87ced7e20d31da1e9f0852cf0a0aa0203039b/core/src/main/java/com/microsoft/applicationinsights/channel/concrete/localforwarder/LocalForwarderTelemetryChannel.java#L22-L24) for details

Reply:
Thanks Updating.

Reply:
done